### PR TITLE
Adding method to get urls to repos in a org

### PR DESCRIPTION
**Pull request by jinder1s**

### CODE REVIEW

File: edx_repo_tools/dev/get_org_repo_names.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"""Clone an entire GitHub organization into current directory."""
```
Comment (nedbat) [inline edx_repo_tools/dev/get_org_repo_names.py:1]: Docstring needs updating.

File: edx_repo_tools/dev/get_org_repo_names.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from edx_repo_tools.auth import pass_github
+
+
+@click.command()
+@click.option(
+    '--forks/--no-forks', is_flag=True, default=False,
+    help="Should forks be included?"
+)
+@click.option(
+    '--depth', type=int, default=0,
```
Comment (nedbat) [inline edx_repo_tools/dev/get_org_repo_names.py:17]: This is unused.

File: edx_repo_tools/dev/get_org_repo_names.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    help="Depth argument for git clone",
+)
+@click.argument(
+    'org'
+)
+@click.option(
+    '--output_file', default="repositories.txt")
```
Comment (nedbat) [inline edx_repo_tools/dev/get_org_repo_names.py:24]: nit: closing paren goes on its own line.

File: edx_repo_tools/dev/get_org_repo_names.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+@pass_github
+def main(hub, forks, depth, org, output_file):
```
Comment (nedbat) [inline edx_repo_tools/dev/get_org_repo_names.py:26]: clone_org.py didn't have one, but it should, and this should have one too: A docstring, which will get printed as part of the --help output.

File: edx_repo_tools/dev/get_org_repo_urls.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+@pass_github
+def main(hub, forks, org, url_type, output_file):
+    """
+    Used to get the urls for all the repositories in a github organization
+    """
+    repositories = []
+    for repo in hub.organization(org).repositories():
+        if repo.fork and not forks:
+            continue
+        if url_type = "ssh":
```
Comment (adzuci) [inline edx_repo_tools/dev/get_org_repo_urls.py:36]: == ?

**Pull request merged**